### PR TITLE
MSTest support for Visual Studio 2015 (Tool ver 14.0)

### PR DESCRIPTION
**Pull request by Morcatko**

### CODE REVIEW

**Comment by patriksvensson** [comment]
Thanks :+1: 
Will include this in the 0.5.1 release.

File: src/Cake.Common.Tests/Unit/Tools/MSTest/MSTestRunnerTests.cs [hunk 1]
```diff
@@ -75,7 +75,8 @@ public void Should_Use_MSTest_From_Tool_Path_If_Provided(string toolPath, string
         }
 
         [Theory]
-        [InlineData("/ProgramFilesX86/Microsoft Visual Studio 12.0/Common7/IDE/mstest.exe")]
+		[InlineData("/ProgramFilesX86/Microsoft Visual Studio 14.0/Common7/IDE/mstest.exe")]
```
Comment (vktr) [inline src/Cake.Common.Tests/Unit/Tools/MSTest/MSTestRunnerTests.cs]: Indentation is off.

File: src/Cake.Common.Tests/Unit/Tools/MSTest/MSTestRunnerTests.cs [hunk 2]
```diff
@@ -75,7 +75,8 @@ public void Should_Use_MSTest_From_Tool_Path_If_Provided(string toolPath, string
...
+		[InlineData("/ProgramFilesX86/Microsoft Visual Studio 12.0/Common7/IDE/mstest.exe")]
```
Comment (vktr) [inline src/Cake.Common.Tests/Unit/Tools/MSTest/MSTestRunnerTests.cs]: Replace tabs with spaces.

**Comment by patriksvensson** [comment]
Merged! :+1:

**Pull request closed without merging**